middleware/router.go

```diff
@@ -53,9 +53,15 @@ func NewRouter(router *gin.Engine) *gin.Engine {
 
 	// HTML Template generated pages
 	router.GET("/status", httpTemplates.StatusPageResponse(), StatusPageMiddleware())
+	router.GET("/console", ConsolePageMiddleware())
+
+	// Authentication
+	router.POST("/signup", api.Signup)
+	router.POST("/signin", api.Signin)
+	router.GET("/signout", api.Signout)
```

Comment:
Using hyphenated routes such as /sign-in, /sign-up, and /sign-out is a matter of style and readability. They are generally more readable and consistent, especially in RESTful API design. Both camelCase and hyphenated routes are acceptable, but hyphenated routes are often preferred for RESTful APIs because they are easier to read.